pkgs/os-specific/linux/rtl8188eus-aircrack/default.nix

```diff
@@ -40,6 +35,6 @@ stdenv.mkDerivation {
     homepage = "https://github.com/aircrack-ng/rtl8188eus";
     license = licenses.gpl2Only;
     maintainers = with maintainers; [ fortuneteller2k ];
-    broken = (lib.versionAtLeast kernel.version "5.17") || ((lib.versions.majorMinor kernel.version) == "5.4" && kernel.isHardened);
+    broken = (lib.versionAtLeast kernel.version "6.6") || ((lib.versions.majorMinor kernel.version) == "5.4" && kernel.isHardened);
```

Comment:
So this kmod cannot be used on e.g., 6.1 LTS? Where did you source that information from?

Reply:
Sorry, I inverted the logic in my head. Of course, it says everything up to and including 6.5 is fine.